python/lsst/ap/association/diaPipe.py

```diff
@@ -114,6 +114,12 @@ class DiaPipelineConnections(
         storageClass="DataFrame",
         dimensions=("instrument", "visit", "detector"),
     )
+    longTrailedSources = pipeBase.connectionTypes.Output(
+        doc="Optional output temporarily storing long trailed diaSources.",
+        dimensions=("instrument", "visit", "detector"),
+        storageClass="DataFrame",
+        name="{fakesType}{coaddName}Diff_longTrailedSrc",
```

Comment:
Could we call it `longTrailedSources` or are you partial to `Src`?

I guess this does parallel `assocDiaSrc` and `diaForcedSrc` etc. so you are just following that... makes me think it should be `longTrlSrc` (don't actually do that)

Reply:
I think I will stick with Src only to mirror the others, but I agree shortening it would be a bad idea.